src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/serializer.html.twig

```diff
@@ -115,20 +115,33 @@
             </div>
 
             <div class="sf-tabs">
-                {{ _self.render_serialize_tab(collector.data, true) }}
-                {{ _self.render_serialize_tab(collector.data, false) }}
-
-                {{ _self.render_normalize_tab(collector.data, true) }}
-                {{ _self.render_normalize_tab(collector.data, false) }}
-
-                {{ _self.render_encode_tab(collector.data, true) }}
-                {{ _self.render_encode_tab(collector.data, false) }}
+                {% for serializer in collector.serializerNames %}
```

Comment:
WebProfilerBundle needs to have a conflict rule ensure it is not used with an older version of the collector

Reply:
Done